### ISSUE TITLE
About LiteFlowNet's pre-trained model as the overpowered teacher in the leakage distillation

### ISSUE THREAD

**Issue by Heroandzhang** [hzwer/arXiv2020-RIFE#221]
您好，有一个问题想要请教一下您：如何在将LiteFlowNet的预训练模型作为overpowered teacher添加到代码中？我没有在代码中找到对LiteFlowNet预训练模型的调用。

**Comment by hzwer** [comment]
你好，这个操作是我们论文的一年前的版本了，当时用的代码是 https://github.com/sniklaus/pytorch-liteflownet
现在建议用 [RAFT](https://github.com/princeton-vl/RAFT)

**Comment by Heroandzhang** [comment]
您好，我在arXiv2020-RIFE/model/IFNet.py 中并没有发现使用超能力教师的代码。
 if gt.shape[1] == 3:
            flow_d, mask_d = self.block_tea(torch.cat((img0, img1, warped_img0, warped_img1, mask, gt), 1), flow, scale=1)
            flow_teacher = flow + flow_d
            warped_img0_teacher = warp(img0, flow_teacher[:, :2])
            warped_img1_teacher = warp(img1, flow_teacher[:, 2:4])
            mask_teacher = torch.sigmoid(mask + mask_d)
            merged_teacher = warped_img0_teacher * mask_teacher + warped_img1_teacher * (1 - mask_teacher)
        else:
            flow_teacher = None
            merged_teacher = None
        for i in range(3):
            merged[i] = merged[i][0] * mask_list[i] + merged[i][1] * (1 - mask_list[i])
            if gt.shape[1] == 3:
                loss_mask = ((merged[i] - gt).abs().mean(1, True) > (merged_teacher - gt).abs().mean(1, True) + 0.01).float().detach()
                loss_distill += ((flow_teacher.detach() - flow_list[i]).abs() * loss_mask).mean()
这段代码看起来像，但是，并没有调用流估计网络的预训练模型。

**Comment by hzwer** [comment]
@Heroandzhang 对，新版论文中已经废弃了用预训练模型的做法；https://arxiv.org/abs/2011.06294

**Comment by Heroandzhang** [comment]
谢谢！！！
